src/Picker/ArticlePickerProvider.php

```diff
@@ -65,6 +88,10 @@ public function getDcaAttributes(PickerConfig $config): array
      */
     public function convertDcaValue(PickerConfig $config, $value): string
     {
+        if ('article' === $config->getContext()) {
+            return (string) $value;
```

Comment:
use `(int)` casting, we're picking an ID (see `PagePickerProvider`). Which means we need to remove the method's return type